assets/sass/_accessibility.scss

```diff
@@ -16,3 +16,16 @@
   position: absolute;
   width: 1px;
 }
+
+/*
+  Warning/notification regarding feature support
+*/
+.legacy-msg {
```

Comment:
‘legacy’ might not be the most apt considering the browser support philosophy — what do you reckon @TrebBrennan ?